core/src/main/java/org/lflang/generator/c/CUtil.java

```diff
@@ -804,4 +807,46 @@ public static String getShortenedName(ReactorInstance instance) {
     }
     return description;
   }
+
+  // Returns the ReactorInstance of the closest enclave in the containment hierarchy.
```

Comment:
Probably all of these method-level comments should be JavaDoc (`/** */`)

Reply:
Yes, thanks!